src/site/content/en/blog/value-of-speed/index.md

```diff
@@ -0,0 +1,200 @@
+Or, if you just want to see how Rel mCvR is calculated, you can get the sample
+data set shown below from
+[this Google Sheet](https://docs.google.com/spreadsheets/d/1Mxmy1luPBOvJneSM9g2NZt6yHKx8xkpBXygOpm4LgNM/edit?usp=sharing)
+and skip steps 1–3.
+
+1. First you'll get the speed and conversion data for the mobile site.
+
+    * Open
+    [this custom report](https://analytics.google.com/analytics/web/template?uid=NPvTU2zoTf2JFsHi6-auyQ)
+    in Google Analytics, select your site from the **Select a view**
+    drop-down list, and click **Create**.
+
+    * Once the report has been generated, click **Select a metric >
+    Metric Group >** **Ecommerce Conversion Rate**.
+    ![image](conversion-rate.jpg)
+    <br><br>
+
+    * Select the **Week** view for the dataset.
+    ![image](landing-pages-week.jpg)
+    <br><br>
+
+    * Select a time period that covers 2–3 months before the speed
+    optimization and 2–3 months after. (In our example, that's January 2018
+    through October 2018.)
+    ![image](analytics-start-date.jpg)
+    <br><br>
+
+    * Export the report to [Google Sheets](http://sheets.google.com) or
+    your preferred spreadsheet application.
+    ![image](export-google-sheets.jpg)
+    <br><br>
+
+1. Now get the speed and conversion data for the desktop site. Open
+[this custom report](https://analytics.google.com/analytics/web/template?uid=X21Nb_soQp69U_ylfNmLvg)
+in Google Analytics and repeat the process for step 1 to export the data to
+a spreadsheet.
+
+1.  Copy the _Ecommerce Conversion Rate_ column in the desktop report
+spreadsheet and insert it next to the _Ecommerce Conversion Rate_ in the
+mobile report spreadsheet.
+
+1.  To calculate the Rel mCvR, create a new column titled _Relative
+Mobile Conversion Rate (Rel mCvR)_. Then add a formula to divide the mobile
+conversion rate by the desktop conversion rate.
+<br><br>
+![image](relative-rate-formula.jpg)
+<br><br>
+
+1. Create a chart based on the _Mobile Avg. Page Load Time (sec)_ and
+_Relative Mobile Conversion Rate (Rel mCvR)_ columns. To do that in Google
+Sheets:
+
+    * Click **Insert > Chart**.
+    * In the **Chart Editor** pane, select the **Line chart** chart type.
+    * Select the _Week Index_, _Mobile Avg. Page Load Time (sec)_, and
+      _Relative Mobile Conversion Rate (Rel mCvR)_ columns as the data ranges.
+    * Click the **Customize** tab and then select **Series**.
+    * In the **Series selector** drop-down list, select **Relative
+      Mobile Conversion Rate (Rel mCvR)**.
+    * In the **Axis** drop-down list select **Right axis**.
+    * If you like, click **Chart & axis titles** to customize the chart title
+      and the left and right _y_-axis labels.
+
+  You should now have a chart that looks something like this:
+  ![image](mobile-versus-relative-chart.jpg)
+  <br><br>
+
+1. Using the chart, identify a period before the speed optimization
```

Comment:
Maybe it's been discussed previously but in this section it might be easier to use regular bullets because the numbering rests to 1

Reply:
@mfriesenhahn Happy to do what you think is best — could you tweak accordingly?

Reply:
I had this issue in another guide as well. When I look at it in another markdown renderer, it's fine. (See [here](https://github.com/mfriesenhahn/markdown-nested-list-test).) Is this a bug?

In any case, there are a couple places where we refer to specific steps (e.g., to identify things you can skip or things you have to repeat). Referring to steps precisely without an ordered list is tricky. The only alternative I can think of is adding sub-section heads. Thoughts?